.github/workflows/code_quality.yml

```
@@ -0,0 +1,60 @@
+name: Code quality checks
+
+# Run this workflow every time a new commit pushed to your repository
+on:
+  push:
+    branches:
+      - master
+      - stable/*
+    tags:
+    paths:
+      - '**.py'
+      - '**.json'
+      - '**.yaml'
+      - '**.in'
+  pull_request:
+    paths:
+      - '**.py'
+      - '**.json'
+      - '**.yaml'
+      - '**.in'
+  workflow_dispatch:
+
+jobs:
+  isort:
+    name: Check import sorting
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
```

```suggestion
      - uses: actions/setup-python@v3
```
v3 is used in other workflow I believe

---

feedback was resolved but I don't see any changes?